source/read.txt

```diff
@@ -11,12 +11,14 @@ Read Data from MongoDB
    :titlesonly:
    :maxdepth: 1
 
+   /read/specify-a-query
```

Comment:
I think this should come after "retrieve" even though you technically have to specify a query to retrieve.  Retrieve still feels like the more common/simple starting point. Not blocking at all and we can work out order of TOC later, just something that popped into my head while looking at this

Reply:
That's a fair point. I'll leave as is for now but we can do a final pass on order in all sections